biz.aQute.repository/src/aQute/maven/provider/POM.java

```diff
@@ -472,13 +486,29 @@ private void getDependencies(Map<Program, Dependency> deps, EnumSet<MavenScope>
 			}
 		}
 
+		if (useDependencyManagement) {
+			for (Map.Entry<Program, Dependency> e : dependencyManagement.entrySet()) {
+				Dependency d = e.getValue();
+
+				if (deps.containsKey(d.program))
+					continue;
+
+				if (scope.contains(d.scope) || (transitive && MavenScope.import_ == d.scope)) {
+					d.bindToVersion(repo);
+					deps.put(e.getKey(), d);
+					if (transitive && d.scope.isTransitive())
```

Comment:
Strangely `MavenScope.import_.isTransitive()` is `false`. I am not sure that is correct. I think we may need to fix that too. Otherwise this will always evaluate to false for import scope.

Reply:
That is true. The test interestingly works regardless of this and I ask myself how, because it shouldn't.